frontend/src/lib/router.tsx

```diff
@@ -723,6 +724,18 @@ const defaultRoutes: {
     disabled: !helpers.isElectron(),
     component: () => <PluginSettings />,
   },
+  pluginDetails: {
+    path: '/settings/plugins/:name',
```

Comment:
I wonder if we will have clashes on the name among diff vendors (but we can solve it later).

Reply:
not only in this instance the clash is possible most of the places where the plugin name is used, would prefer to solve it later as you mentioned.